### PR TITLE
 tmz refactor punchtape

### PR DESCRIPTION
**Pull request by t-m-z**

### CODE REVIEW

File: query [hunk 1]
```diff
@@ -0,0 +1 @@
+aehd
```
Comment (Mike-3) [inline query:1]: Neccesary ?

File: lib/application/i18n/assets/en.json [hunk 2]
```diff
@@ -3995,8 +3995,19 @@
   "punchtape_example": "GCW → [OO .O ] [ OO.O ] [O  .OO]",
   "punchtape_ccitt_title" : "Punch Tape Telegraph (Baudot/Murray/CCITT et al)",
   "punchtape_ccitt_description" : "Electrical Telegraph with 5 Bit Punch Tape invented by Emile Baudot and Donald Murray, normed by CCITT",
-  "punchtape_mode_binary" : "Mirrored\nBit-Order 54321",
-  "punchtape_mode_original" : "Original\nBit-Order 12345",
+  "punchtape_mode_numberonly" : "Only Numbers",
+  "punchtape_mode_lettersandnumbers" : "Letters & Numbers",
+  "punchtape_mode_baudot" : "Baudot",
+  "punchtape_mode_bitorder" : "Bit Order",
+  "punchtape_mode_bitorder_54123" : "54 123",
```
Comment (Mike-3) [inline lib/application/i18n/assets/en.json:4002]: Is this right with space ?

File: lib/application/registry.dart [hunk 3]
```diff
@@ -840,16 +840,6 @@ void initializeRegistry(BuildContext context) {
       ],
       licenses: const [],
     ),
-
-    GCWTool(
```
Comment (Mike-3) [inline lib/application/registry.dart:844]: Delete Exif Reader ?

**Review by Mike-3 — CHANGES_REQUESTED** [review]

File: lib/application/registry.dart (hunk 3 above)
Comment (t-m-z) [reply lib/application/registry.dart:844]: I found ExifReader twice in the Registry

File: query (hunk 1 above)
Comment (t-m-z) [reply query:1]: not really - I have no clue where this file came from

File: lib/application/i18n/assets/en.json (hunk 2 above)
Comment (t-m-z) [reply lib/application/i18n/assets/en.json:4002]: I will change it to Bit Sequence

File: lib/application/registry.dart (hunk 3 above)
Comment (t-m-z) [reply lib/application/registry.dart:844]: The entry was twice

**Pull request merged**